features/telemetry/protocols/netflow/adapter/src/main/java/org/opennms/netmgt/telemetry/protocols/netflow/adapter/common/NetflowMessage.java

```diff
@@ -110,6 +110,9 @@ public Integer getEngineType() {
 
     @Override
     public Long getDeltaSwitched() {
+        if (getNetflowVersion().equals(NetflowVersion.V5)) {
+            return flowMessageProto.hasFirstSwitched() ? flowMessageProto.getFirstSwitched().getValue() : null;
```

Comment:
Maybe we can always use this as an fallback for `deltaSwitchen` without making it protocol specific

Reply:
Yeah, If you think that would be correct protocol wise, I can change that.